File: Domain/Entities/Person.cs
using Domain.Enums;
using System;

namespace Domain.Entities
{
    public class Person
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string LastName { get; set; }

        public Sex Sex { get; set; }

        public DateTime? BirthDate { get; set; }
    }
}

File: WebUI/ClientApp/src/app/core/models/person.ts
import { Sex } from 'src/app/core/enums/sex';

export class Person {
  private _birthDate: Date;

  public set birthDate(value : Date) {
    this._birthDate = !value ? null : value;
  }
  
  public get birthDate() : Date {
    return this._birthDate;
  }
  
  constructor(
    public id: number = 0,
    public name: string = '',
    public lastName: string = '',
    public sex: Sex = Sex.Male,
    birthDate: Date = null
  ) { 
    this.birthDate = birthDate;
  }
}